### PR TITLE
onSubmit is called twice when pressing enter in EditableTitle

### PR DESCRIPTION
**Pull request by aryelu**
### 🔦 Summary
...

### ✅ Checklist
- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

File: src/EditableTitle/EditableTitle.js [hunk 1]
```diff
@@ -95,13 +95,17 @@ class EditableTitle extends React.Component {
...
-            onEnterPressed={this.onValueSubmission}
+            onEnterPressed={this.onEnterPressed}
           />
         </div>
       </div>
     );
   }
 
+  onEnterPressed = () => {
+    this.wsrInput.blur();
```
Comment (mykas) [inline src/EditableTitle/EditableTitle.js]: So this make input blurred but the state of the component is still gonna be `focus: true`. Is this the use case?

**Comment by shlomitc** [comment]
@aryelu - I think it's time to add a test for the `enter` key press behavior, to prevent this from recurring

File: src/EditableTitle/EditableTitle.js (hunk 1 above)
Comment (aryelu) [reply src/EditableTitle/EditableTitle.js]: the blur will be handled by onBlur

File: src/Input/Input.uni.driver.js [hunk 2]
```diff
@@ -75,7 +75,7 @@ export const testkit = base => {
     focus: async () => await reactBaseInput.focus(),
     blur: async () => await reactBaseInput.blur(),
     keyUp: async () => await reactBaseInput.keyup(),
-    keyDown: async () => await reactBaseInput.keydown(),
+    keyDown: async key => await reactBaseInput.keydown(key),
```
Comment (shlomitc) [inline src/Input/Input.uni.driver.js]: please name `key` as `eventData`

Comment (aryelu) [reply src/Input/Input.uni.driver.js]: 👍

**Pull request merged**